index.bs

```diff
@@ -184,12 +179,17 @@ Integration with HTML {#html}
 
 This specification integrates with the [[!HTML]] specification by patching the algorithms below:
 
-Document object initialization {#document-init}
+Navigation fetch {#navigation-fetch}
+----------
+
+At <a href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#process-a-navigate-fetch">process a navigate fetch</a>,
```

Comment:
This `dfn` should be exported, can we use bikeshed syntax for auto linking? 

https://tabatkins.github.io/bikeshed/#autolink-microsyntax

(same for "process a navigate response")